AgentDiscoveries-Frontend/app/src/components/todays-code-page.jsx

```diff
@@ -22,12 +22,22 @@ export default class TodaysCodePage extends React.Component {
         this.slideDown = this.slideDown.bind(this);
     }
 
+    componentDidMount() {
+        $('#react-root').css('background-color', 'rgba(0, 0, 0, 0.7)');
+        $('.opening-top').animate({ width: '0%' }, 800, function() {});
+        $('.opening-bottom').animate({ width: '0%' }, 800, function() {});
+    }
+
     render() {
-        console.log(this.state.showMessage);
+
         return (
             <React.Fragment>
 
+                <div className='opening-top'></div>
```

Comment:
Discussed offline with Dan - I think this is being solved the wrong way, with the login page & message pages explicitly caring about the react-root, but given the time I think we're better off leaving this now